addon/components/nav-modal.js

```diff
@@ -31,23 +31,23 @@ export default Component.extend({
     'frostNavigation.categories',
     'activeCategory',
     function () {
-      if (!this.get('activeCategory')) {
-        return false
-      }
       const categories = this.get('frostNavigation.categories') || A()
       const category = categories.find((category) => {
         let name = category.name.toLowerCase()
         return name === this.get('activeCategory').toLowerCase()
       })
       return category ? category.columns : null
-    }),
+    }
+  ),
+  init () {
+    this._super(...arguments)
+    ;['outsideClick', 'escape'].forEach(e => {
+      this.actions[e] = function () {
+        this.get('frostNavigation').dismiss()
+      }
+    })
```

Comment:
For lines 44-48, you will run into problems trying to test this code inside init()… Instead, move that into a function that you call inside init(). Then you can call/stub that method to test that it is doing what you expect.